runProcessor.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+nvm use
+yarn install
+yarn build
+docker-compose up -d db-kintsugi-testnet
+yarn db:migrate
```

Comment:
Does this work if the container has been stopped before?
I'm wondering if it needs to wait for docker to spin up the db before attempting `db:migrate`

You could eg. start the db first before doing `nvm use` through to `yarn build` . That might buy enough time more reliably before trying to run the migration.

Reply:
Or find a way to wait for the container to have been started before continuing - which would be the best way to make sure we can run migrate.

Reply:
Ahhh switching the order is clever. I will make that change.

Reply:
I made the change you requested and added a fun box to allow users to choose their network!

Reply:
> I made the change you requested and added a fun box to allow users to choose their network!

I like the idea. But, during startup the .env file dictates which network the processor is expecting to be run.
Can we use that to our advantage in this script? Something to think about.

Reply:
Great idea! I adding functionality to script to automatically set the environment variables based on the chosen network.

Reply:
> Great idea! I adding functionality to script to automatically set the environment variables based on the chosen network.

I meant the other way around:
Read the env variables and then the script selects the db to start based on what has been set, eg. in ARCHIVE_ENDPOINT

Reply:
Ah I see what you mean. I changed the functionality to work the other way around